client/app/actions/index.js

```diff
@@ -35,6 +42,35 @@ export const fetchMatchesAsync = userId => {
   };
 };
 
+const fetchProfile = profile => ({
+  type: FETCH_PROFILE,
+  profile
+})
+
+export const fetchProfileAsync = userId => {
+  return async dispatch => {
+    let profile;
+
+    if (useMocks) {
+      const uri = 'https://meetover.herokuapp.com/test/liprofile';
+      const init = { method: 'POST' };
+
+      const response = await fetch(uri, init);
+      profile = await response.json();
+      profile = JSON.parse(profile);
```

Comment:
Is this necessary?

Reply:
Yes. Talked offline. This is a server side problem, can likely be addressed with the content type